smart-contracts/contracts/wallet/QuickWallet.sol

```diff
@@ -43,20 +44,20 @@ contract QuickWallet {
         )).toEthSignedMessageHash().recover(txSignature);
         require(owner() == _signer, "QuickWallet: Signer is not wallet owner");
 
-        _call(receiver, data);
+        _call(receiver, data, value);
 
         if (feeValue > 0) {
           bytes memory feePaymentData = abi.encodeWithSelector(
               bytes4(keccak256("transfer(address,uint256)")), feeReceiver, feeValue
           );
-          _call(feeToken, feePaymentData);
+          _call(feeToken, feePaymentData, 0);
         }
 
         txCount++;
```

Comment:
I think this should be moved up to above line 47 to prevent attackers from replaying the transactions by reentering from the call on line 47 or 53.